frontend/src/app/shared/order-button-group/order-button-group.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, EventEmitter, Input, Output } from "@angular/core";
+import { actionIcons } from "@shared/icons";
+
+export type OrderChange = "up" | "down";
+
+@Component({
+    selector: "lc-order-button-group",
+    templateUrl: "./order-button-group.component.html",
+    styleUrls: ["./order-button-group.component.scss"],
+})
+export class OrderButtonGroupComponent {
+    @Input() entityName: string | null = null;
```

Comment:
As far as I know, there is no reason to make this configurable, at least in the foreseeable future. You can just write "episode" in the template.